services/postgres/Makefile

```diff
@@ -0,0 +1,57 @@
+.DEFAULT_GOAL := help
+
+REPO_BASE_DIR  = $(abspath $(CURDIR)/../../../)
+VENV_DIR ?= $(abspath $(REPO_BASE_DIR)/.venv)
+
+APP_NAME     := $(notdir $(CURDIR))
```

Comment:
As an devop convention, the name of the folder is the name of the service you run ... this supports the idea of renaming folder to something other than ``postgres`` :-)